performance_test/helper_scripts/apex_performance_plotter/apex_performance_plotter/generate_plots.py

```diff
@@ -24,6 +24,7 @@ def generate_figures(xaxis,
                      minflt_ydata,
                      majflt_ydata,
                      nivcsw_ydata,
+                     cpusage_ydata,
```

Comment:
maybe cpu_usage_ydata? Just a suggestion, but now you kind of have cp usage (the same line 67 and 70 in this file)

Reply:
Okay changing to `cpu_usage_ydata` everywhere